pyeudiw/tests/settings.py

```diff
@@ -233,6 +234,33 @@
                 },
             },
         },
+        "x509": {
+            "module": "pyeudiw.trust.handler.x509",
+            "class": "X509Hanlder",
+            "config": {
+                "client_id": f"{BASE_URL}/OpenID4VP",
+                "relying_party_certificate_chains_by_ca":{
+                    "ca.example.com": gen_chain(leaf_cn="example.com"),
```

Comment:
NO

if rp's client id is X, leaf_cn must be X, otherwise an exception durint the init must thrown

Reply:
if client id scheme is x509 san dns, the check must be on the san dns name and the client_id must have the same value of the certificate's san dns name

if client id scheme is x509 san uri, the check must be on the certificateìs san uris

if the certificate does not have any san, the dns name must be checked upon the CN

Reply:
@PascalDR leaf's CN would be the entire HTTPs URL and not the dns name alone

please with this since it seems misleading